### PR TITLE
feat(blog): we have been busy blog post

### PR DESCRIPTION
**Pull request by janl**
I’d like to post this on Tuesday, Nov. 17, ~11am CET.

I have prepared three follow up posts for the respective next Tuesdays on the topics.

### CODE REVIEW

File: _posts/2015-11-17-we-have-been-busy.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+*If you want to get involved with the publication process on the Hoodie blog, or the project in general: [we are actively recruiting and editorial team](https://github.com/hoodiehq/discussion/issues/84).*
+
+Here are the highlights
+
+## 1. Hoodie 1.0 milestones closing in
+
+We made some **significant progress** on [the Hoodie 1.0 milestones](http://gr2m.github.io/milestones/). Most notably, we [completely re-structured how Hoodie is put together, and tested and released](https://github.com/hoodiehq/discussion/issues/76). This work is already available in our new @next tag on npm, albeit not 100% functional yet. If you are keen on giving it a spin and help fix missing features, run *npm install hoodie@next* to get the work-in-progress version. See [https://github.com/hoodiehq/hoodie](https://github.com/hoodiehq/hoodie) for more details and a a way to find and file issues. The main work is done in the [Client](https://github.com/hoodiehq/hoodie-client), [Server](https://github.com/hoodiehq/hoodie-server) and [Admin Dashboard](https://github.com/hoodiehq/hoodie-admin-dashboard) modules.
```
Comment (varjmes) [inline _posts/2015-11-17-we-have-been-busy.md]: "completely re-structured how Hoodie is put together, tested and released" flows a little better IMO

Comment (varjmes) [inline _posts/2015-11-17-we-have-been-busy.md]: are we able to link to "@next" in some way?

**Comment by verpixelt** [comment]
I din't know there was a "team" picture 😪

**Comment by janl** [comment]
@verpixelt we couldn’t find everyone in time :/

**Comment by verpixelt** [comment]
I'm proposing this as an addition:
https://flic.kr/p/yfMtR4 😜
(@janl all good n fine ;))

File: _posts/2015-11-17-we-have-been-busy.md (hunk 1 above)
Comment (janl) [inline _posts/2015-11-17-we-have-been-busy.md]: @Charlotteis we could link to https://github.com/hoodiehq/hoodie-server/tree/next would that help?

**Comment by janl** [comment]
@verpixelt can you photoshop that into the group pic? :P

File: _posts/2015-11-17-we-have-been-busy.md (hunk 1 above)
Comment (varjmes) [inline _posts/2015-11-17-we-have-been-busy.md]: @janl yep!

**Comment by janl** [comment]
@boennemann lol thanks, that draft has been a bit old :D

**Comment by verpixelt** [comment]
@janl totally! Give me a sec!

**Comment by varjmes** [comment]
Pls photoshop my face in

**Comment by janl** [comment]
pushed a new version with all the suggestions, thanks!

**Comment by verpixelt** [comment]
@janl good, eh?! ;P
https://www.dropbox.com/s/l5pmg4ji8iw7p9v/jsconf.gif?dl=0

**Comment by NickColley** [comment]
ship it :ship:

**Comment by janl** [comment]
ROFLMAO <3

Now, how to do a hover-effect on our blog? Would totally swap in your image on mouseover ;)

**Comment by varjmes** [comment]
i approve of this gif

**Comment by gr2m** [comment]
HAHAHAHHAHAHA

**Comment by janl** [comment]
And we’re live, thanks all!

**Pull request merged**